tests/integration/conftest.py

```diff
@@ -33,7 +33,7 @@ def run_and_rollback(tx, cypher, **parameters):
         raise ForcedRollback(value)
 
     def f(cypher, **parameters):
-        with bolt_driver.session() as session:
+        with driver.session() as session:
```

Comment:
Why did it changed?

Reply:
The fixture was broken. `bolt_driver` is an undefined variable here. It just went unnoticed because this fixture is currently unused. But I don't want to get rid of it as it's a very handy one that I use every now and then (like I did while developing this PR, that's when I noticed). I also can't rule out that I might use this fixture for some tests at a later point.